spec/jobs/s3_west_delivery_job_spec.rb

```diff
@@ -4,46 +4,44 @@
   let(:druid) { 'bj102hs9687' }
   let(:version) { 1 }
   let(:dvz) { DruidVersionZip.new(druid, version) }
+  let(:dvz_part) { DruidVersionZipPart.new(dvz, part_s3_key) }
   let(:object) { instance_double(Aws::S3::Object, exists?: false, put: true) }
   let(:bucket) { instance_double(Aws::S3::Bucket, object: object) }
   let(:md5) { '4f98f59e877ecb84ff75ef0fab45bac5' }
-  let(:metadata) { dvz.metadata.merge(zip_version: 'Zip 3.0 (July 5th 2008)') }
+  let(:base64) { dvz.hex_to_base64(md5) }
+  let(:metadata) { dvz_part.metadata.merge(zip_version: 'Zip 3.0 (July 5th 2008)') }
+  let(:part_s3_key) { dvz.s3_key('.zip') }
 
   before do
     allow(Settings).to receive(:zip_storage).and_return(Rails.root.join('spec', 'fixtures', 'zip_storage'))
     allow(PreservationCatalog::S3).to receive(:bucket).and_return(bucket)
     allow(ResultsRecorderJob).to receive(:perform_later).with(any_args)
   end
 
-  it 'descends from DruidVersionJobBase' do
-    expect(described_class.new).to be_an(DruidVersionJobBase)
-  end
-
-  it 'populates a DruidVersionZip' do
-    expect(DruidVersionZip).to receive(:new).with(druid, version).and_return(dvz)
-    described_class.perform_now(druid, version, metadata)
+  it 'descends from ZipPartJobBase' do
+    expect(described_class.new).to be_an(ZipPartJobBase)
   end
 
   context 'zip already exists on s3' do
```

Comment:
in truth, we are now talking about a zip-part, not a zip, yes?   What is the suffix of the file being replicated?